guide/Text/Embed/example.md

```diff
@@ -1,3 +1,46 @@
 # Complete Embed
-Pls check this Embed functions first
-...
\ No newline at end of file
+
+Here is a code example of a complete embed.
+
+## Function Format
+
+$title[Title;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$author[Custom Command;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp;https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$image[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$description[Description]
+$thumbnail[https://cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp]
+$color[#0c2238]
+$footer[Footer]
+
+## Embed Format
+
+{title:Title}
+{url:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+{footer:Footer}
+{description:Description}
+{color:#0c2238}
+{author:Custom Command:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+{thumbnail:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+{image:https#COLON#//cdn.discordapp.com/icons/832255686783533066/f7131f694c6e1a2bd9c360d8b525d4e3.webp}
+
```

Comment:
This list is missing lot of things, check the curl reference page to populate it

Reply:
couldn't find curl reference page, so i hope theres enough